chains/metertestnet/logo.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ viewBox="0 0 64.000000 64.000000"
+ preserveAspectRatio="xMidYMid meet">
+
+<g transform="translate(0.000000,64.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M275 590 c-61 -10 -91 -26 -139 -70 -65 -60 -94 -134 -87 -224 2 -37
+37 10 50 68 46 204 291 247 416 73 14 -20 26 -26 38 -21 27 10 21 28 -28 82
+-68 76 -154 107 -250 92z"/>
+<path d="M176 344 c-48 -41 -102 -86 -120 -100 -23 -19 -8 -14 61 20 51 25 94
+46 96 46 2 0 1 -14 -1 -32 l-5 -32 49 29 c27 16 64 38 83 49 37 22 35 26 16
+-41 -3 -13 -4 -23 0 -23 8 0 245 131 245 135 0 2 -39 -9 -87 -26 -48 -16 -89
+-29 -91 -29 -3 0 -2 18 2 40 3 22 3 40 0 40 -4 0 -40 -18 -81 -41 l-74 -41 7
+41 c4 23 3 41 -2 40 -5 0 -49 -34 -98 -75z"/>
+<path d="M568 337 c-9 -6 -21 -35 -28 -64 -45 -198 -294 -243 -412 -75 -12 18
+-22 22 -37 17 -12 -3 -21 -10 -21 -15 0 -4 20 -30 45 -58 60 -66 119 -94 205
+-94 83 0 143 27 199 87 45 49 65 91 73 151 7 51 -2 69 -24 51z"/>
+</g>
+</svg>
+ 
```

Comment:
only need svg tags + minified on https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 64 64"><path d="M27.5 5c-6.1 1-9.1 2.6-13.9 7-6.5 6-9.4 13.4-8.7 22.4.2 3.7 3.7-1 5-6.8C14.5 7.2 39 2.9 51.5 20.3c1.4 2 2.6 2.6 3.8 2.1 2.7-1 2.1-2.8-2.8-8.2-6.8-7.6-15.4-10.7-25-9.2z"/><path d="M17.6 29.6c-4.8 4.1-10.2 8.6-12 10-2.3 1.9-.8 1.4 6.1-2 5.1-2.5 9.4-4.6 9.6-4.6.2 0 .1 1.4-.1 3.2l-.5 3.2 4.9-2.9c2.7-1.6 6.4-3.8 8.3-4.9 3.7-2.2 3.5-2.6 1.6 4.1-.3 1.3-.4 2.3 0 2.3.8 0 24.5-13.1 24.5-13.5 0-.2-3.9.9-8.7 2.6-4.8 1.6-8.9 2.9-9.1 2.9-.3 0-.2-1.8.2-4 .3-2.2.3-4 0-4-.4 0-4 1.8-8.1 4.1l-7.4 4.1.7-4.1c.4-2.3.3-4.1-.2-4-.5 0-4.9 3.4-9.8 7.5z"/><path d="M56.8 30.3c-.9.6-2.1 3.5-2.8 6.4-4.5 19.8-29.4 24.3-41.2 7.5-1.2-1.8-2.2-2.2-3.7-1.7-1.2.3-2.1 1-2.1 1.5 0 .4 2 3 4.5 5.8 6 6.6 11.9 9.4 20.5 9.4 8.3 0 14.3-2.7 19.9-8.7 4.5-4.9 6.5-9.1 7.3-15.1.7-5.1-.2-6.9-2.4-5.1z"/></svg>
```

Reply:
Thanks, I had troubles with this svg file.